### PR TITLE
Upgraded quickstart-example with hbase-1.9.0 version.

### PR DESCRIPTION
**Pull request by rahulKQL**
* Updated `bigtable-hbase-1.x` to 1.9.0.
* Added more formatted terminal comments.
* Standardized `projectID` argument for Integration Test.

### CODE REVIEW

File: java/quickstart/pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@ limitations under the License.
     <dependency>
       <groupId>com.google.cloud.bigtable</groupId>
       <artifactId>bigtable-hbase-1.x</artifactId>
-      <version>1.4.0</version>
+      <version>1.9.0</version>
```
Comment (jaceklaskowski) [inline java/quickstart/pom.xml]: Why not `2.x` then? Just curious.

Comment (rahulKQL) [reply java/quickstart/pom.xml]: We support both `hbase-1.x` & `2.x` version and would continue it for at least next few quarters, So I tried not to change the original intent for these examples.

**Comment by rahulKQL** [comment]
@sduskis Could you please have look on these change

File: java/quickstart/pom.xml (hunk 1 above)
Comment (sduskis) [reply java/quickstart/pom.xml]: We should continue with 1.x for now.  Most of our users use 1.x.

**Pull request merged**